### PR TITLE
Feat: Update Wizard Logic to Display Cancel Button and Info Copy

### PR DESCRIPTION
**Pull request by erikataot**
# Description
- During company claim, the Wizard (page 2) was missing the `Cancel` button
- The page was also missing the info box about all required fields

BEFORE
Screenshot 2024-03-14 141152

AFTER
Screenshot 2024-03-14 142542

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1267-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1267-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1267-dops.apps.silver.devops.gov.bc.ca/api)
- [TPS-Migration](https://onroutebc-1267-tps-migration.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/features/wizard/subcomponents/CompanyAndUserInfoSteps.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const CompanyAndUserInfoSteps = ({
   return (
     <>
       <input type="hidden" {...register("legalName")} />
-      {activeStep === totalSteps - 2 && (
+      {activeStep <= totalSteps - 1 && (
```
Comment (krishnan-aot) [inline frontend/src/features/wizard/subcomponents/CompanyAndUserInfoSteps.tsx:120]: Why this change?

Comment (erikataot) [reply frontend/src/features/wizard/subcomponents/CompanyAndUserInfoSteps.tsx:120]: without it the info text banner doesn't show up on that page (when it's in the Hi-defs)

image

Comment (krishnan-aot) [reply frontend/src/features/wizard/subcomponents/CompanyAndUserInfoSteps.tsx:120]: Can you share the link to hifi?

**Review by krishnan-aot — APPROVED** [review]
